bookshelf/gce/deploy.sh

```diff
@@ -0,0 +1,151 @@
+# Copyright 2015 Google Inc.
+#d
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#! /bin/bash
+
+set -ex
+
+ZONE=us-central1-f
+
+GROUP=frontend-group
+TEMPLATE=$GROUP-tmpl
+MACHINE_TYPE=f1-micro
```

Comment:
It's the same one we are using for the rest of the tutorial. Presumably because it's cheapest.

And for the size of this app, it runs just fine :smile: